tools/invoke-tests

```diff
@@ -52,5 +52,10 @@ export OS_AUTOINST_BUILD_DIRECTORY=$build_directory
 export OS_AUTOINST_MAKE_TOOL=$make_path
 
 # invoke tests within the t and xt directory
-cd t && "$prove_path" $PROVE_ARGS ${TESTS:-'-r'}
-cd ../xt && "$prove_path" $PROVE_ARGS ${TESTS:-'-r'}
+if [ -z $TESTS ]
```

Comment:
`-z` is like a negative check so we have double negation. I suggest to turn that around and use `-n` and use the current "else"-path first

Reply:
Thank you for the suggestion, I could in the same way not use -`z` or `-n` at all ( `[ $TESTS]` or `[[  $TESTS ]]` would work fine). However, may I suggest `[[ $TESTS = *[!\ ]* ]]`? Unlike all other options, it will behave correctly even in the case the user provides an empty or whitespace $TESTS variable.